samples/msal-react-samples/typescript-sample/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "typescript-sample",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/msal-browser": "file:../../../lib/msal-browser",
+    "@azure/msal-react": "file:../../../lib/msal-react",
+    "@types/node": "^12.19.4",
+    "@types/react": "^16.9.56",
+    "@types/react-dom": "^16.9.9",
+    "bootstrap": "^4.5.3",
+    "react": "file:../../../lib/msal-react/node_modules/react",
+    "react-bootstrap": "^1.4.0",
+    "react-dom": "file:../../../lib/msal-react/node_modules/react-dom",
+    "react-scripts": "3.4.3",
```

Comment:
May be worth upgrading to v4?

Reply:
Can also be done later.

Reply:
Was having problems with bootstrap + react-scripts v4 + importing react/react-dom locally.

I'm thinking about switching all the react samples over to MaterialUI or FluentUI and will upgrade then.